src/HTTP/Request.ts

```diff
@@ -65,6 +66,17 @@ export interface RetrievalPreferences {
 type ResolveCallback = ( response:Response ) => void;
 type RejectCallback = ( error:HTTPError ) => void;
 
+function __addErrors<T extends {}>( o:T ):void {
+	Object
+	.keys( o )
+	.map( k => o[ k ] )
+	.filter( e => {
+		if (e.statusCode === null || e instanceof Map) return false;
+		statusCodeMap.set( e.statusCode, e );
+	} );
+}
+__addErrors( Errors );
+
```

Comment:
There is no need the use a function if it's only called when the module is created.